.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,21 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: false
+contact_links:
+- name: Create an **announcement**
+  url: https://github.com/ansible-collections/news-for-maintainers/issues/new
+  about: |
+    Use issues in this repositories only for announcements to collection maintainers.
+    By creating an issue you will spam a large amount of folks who will be annoyed if you post anything but a relevant announcement.
+    So **do not use this for anything but appropriate announcements!**
+- name: '**Discuss** an announcement'
```

How about aligning the style with other menu items?
```suggestion
- name: 💬 **Discuss** an announcement
```